cinema/serializers.py

```diff
@@ -42,17 +65,41 @@
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors"
+        )
 
 
 class MovieSessionSerializer(serializers.ModelSerializer):
+    tickets_available = serializers.SerializerMethodField()
```

Comment:
Ensure that the logic in `get_tickets_available` correctly reflects the business rules for ticket availability. The current implementation assumes all seats are available unless taken, which may not account for reserved or blocked seats.